pkg/models/user_token.go

```diff
@@ -1,81 +1,9 @@
 package models
 
-import (
-	"context"
-	"errors"
-	"net"
-
-	"github.com/grafana/grafana/pkg/registry"
-	"github.com/grafana/grafana/pkg/services/user"
-)
-
-// Typed errors
-var (
-	ErrUserTokenNotFound = errors.New("user token not found")
-)
-
-// CreateTokenErr represents a token creation error; used in Enterprise
-type CreateTokenErr struct {
-	StatusCode  int
-	InternalErr error
-	ExternalErr string
-}
-
-func (e *CreateTokenErr) Error() string {
-	if e.InternalErr != nil {
-		return e.InternalErr.Error()
-	}
-	return "failed to create token"
-}
-
-type TokenExpiredError struct {
-	UserID  int64
-	TokenID int64
-}
-
-func (e *TokenExpiredError) Error() string { return "user token expired" }
-
 type TokenRevokedError struct {
 	UserID                int64
 	TokenID               int64
 	MaxConcurrentSessions int64
 }
 
 func (e *TokenRevokedError) Error() string { return "user token revoked" }
```

Comment:
question: why are these not moved over as well?

Reply:
Moving this one causes circular dependency. Fixed that by creating a sub package and alias. Also had to move UserToken to that package as well